src/signals/incident-management/components/CalendarInput/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+import CustomInput from './CustomInput';
+
+const CalendarInput = ({ id, label, name, onSelect, selectedDate }) => (
+  <Fragment>
+    <DatePicker
+      autoComplete="off"
+      customInput={<CustomInput label={label} id={id} />}
+      dateFormat="DD-MM-YYYY"
+      id={id}
```

Comment:
Will these components have the same `id`?

Reply:
Good catch. The `DatePicker` component doesn't do anything with the `id` prop. I'll remove it.

Reply:
It does do something? :)

Reply:
Well, I removed it and then realised I use it in the tests. So I added it back
🤷‍♂